docs/modules/ROOT/pages/tutorial-kubernetes.adoc

```diff
@@ -1,73 +1,171 @@
-= Kubernetes example
+= CMOS Kubernetes Deployment
 
 [abstract]
-A simple working example from scratch locally using https://kind.sigs.k8s.io/[Kubernetes-IN-Docker (KIND)^] with the https://docs.couchbase.com/operator/current/overview.html[Couchbase Autonomous Operator (CAO)^] deploying Couchbase Server then being monitored by CMOS.
+A simple working example from scratch in kubernetes cluster with the https://docs.couchbase.com/operator/current/overview.html[Couchbase Autonomous Operator (CAO)^] deploying Couchbase Server then being monitored by CMOS.
 
 include::partial$tutorial.adoc[]
 
+== Overview
+
+CMOS is a simple, out-of-the-box solution based on industry standard tooling to observe the state of your Couchbase cluster.
```

Comment:
Add a back link to architecture page

Reply:
Second para:
CMOS can be deployed to monitor the CB clusters deployed via Couchbase Autonomous Operator.